src/debug_info.rs

```diff
@@ -39,7 +39,7 @@ impl DebugInfo {
                     .map(|x| x.map(|location| (type_declaration.id.clone(), location)))
                     .transpose()
             })
-            .collect::<Result<HashMap<_, _>, DiagnosticAdded>>()?;
```

Comment:
can you re-add this one and the ones that have explicit types?
It is useful to know the types of the variables (when reading the code without editor support)

Reply:
Are you sure because this is explicit by reading the function definition a few lines before no? Implicitly inferred, maybe I missed something

Reply:
I think it is better to leave the `.collect::<Result<HashMap<_, _>, DiagnosticAdded>>()?`
Aldo the types are inferred, its make the code more readable